initializers/sequelize.js

```diff
@@ -99,3 +118,62 @@ module.exports = {
     });
   }
 };
+
+function migrateSequelizeMeta(api, umzug) {
```

Comment:
It doesn't make a backup of the existing SequelizeMeta table, and it adds the completed migrations based on the files in the migrations directory - I can see it failing for the case where you:

1. Have the old SequelizeMeta table
2. Have new migration files added

Come to think of it - since this is going to be a one-time operation, would it be better to detect the old table and error out, asking the user to run `sequelize db:migrate:old_schema`? Better than having half-working code here.